src/igvfd/schemas/changelogs/modification.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`modification.json`*
 
+### Schema version 6
+* Adjust `fused_domain` enum list to remove `KRAB`.
```

Comment:
Extra blank line needed above this line.

Reply:
done